cli/sandbox.go

```diff
@@ -162,8 +162,13 @@ func destroySandbox(vm *hypervisor.Vm, lockFile *os.File) {
 
 	unlockSandbox(lockFile)
 
-	if err := os.RemoveAll(sandboxPath(vm)); err != nil {
-		glog.Errorf("can't remove vm dir %q: %v", filepath.Join(hypervisor.BaseDir, vm.Id), err)
+	// Perform rmdir on sharefs at first, don't recursively remove it in case the rootfs container was failed to be umounted
+	if err := os.Remove(filepath.Join(sandboxPath(vm), hypervisor.ShareDirTag)); err != nil {
+		glog.Errorf("can't remove shared dir %q: %v", filepath.Join(sandboxPath(vm), hypervisor.ShareDirTag), err)
```

Comment:
There is `rootfs` directory in sandbox/shardir so this removal will fail. And it does fail and causes CI failure.